packages/http/configuration-schema.json

```diff
@@ -46,5 +46,8 @@
     }
   },
   "type": "object",
-  "additionalProperties": true
-}
+  "additionalProperties": true,
+  "required": [
+    "baseUrl"
```

Comment:
baseUrl isn't required though? :thinking:

Reply:
I think all our adaptor should at-least have on required property because. We want to display credential based on required properties.

Reply:
We shouldn't force required properties if they're not needed, even for the sake of consistency. It's annoying and misleading.

What do you mean "displaying the credential"? Where are we displaying it? You mean in Lightning?

Reply:
@josephjclark i have removed the required for `http`. Please review this PR again

Reply:
I agree this should be fixed on `docs` side not adaptors